### PR TITLE
Validation: Added msauthor and Table Headers

### PR DESCRIPTION
**Pull request by kturetz**

### CODE REVIEW

**Comment by kturetz** [comment]
@stevemunk 
@rbrundritt 
@DavidBuerer
@yasong-Ashley 
@jesselgit 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: BingMaps/styling/map-style-sheet-entries.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
 ---
-title: "Map Style Sheet Entries | Microsoft Docs"
-ms.custom: ""
-ms.date: "05/26/2020"
-ms.reviewer: ""
-ms.suite: ""
-ms.tgt_pltfrm: ""
-ms.topic: "article"
+title: Map Style Sheet Entries | Microsoft Docs
+description: The entries listed below can be used in a map style sheet to customize the appearance of a map in various Microsoft map controls.
```
Comment (stevemunk) [inline BingMaps/styling/map-style-sheet-entries.md:3]:
```suggestion
description: A list of settings and elements that can be used in a map style sheet to customize the appearance of a map in various Microsoft map controls. They can also be used to define how overlaid geometry on a map should be represented.
```

File: BingMaps/styling/map-style-sheet-entry-properties.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
 ---
-title: "Map Style Sheet Entry Properites | Microsoft Docs"
-ms.custom: ""
-ms.date: "05/26/2020"
-ms.reviewer: ""
-ms.suite: ""
-ms.tgt_pltfrm: ""
-ms.topic: "article"
+title: Map Style Sheet Entry Properties | Microsoft Docs
+description: The properties listed below can be set on entries in a map style sheet to customize the appearance of a map in various Microsoft map controls.
```
Comment (stevemunk) [inline BingMaps/styling/map-style-sheet-entry-properties.md:3]:
```suggestion
description: Learn about the map style sheet entry properties that can be set on entries in a map style sheet to customize the maps appearance in various Microsoft map controls.
```

File: BingMaps/styling/map-style-sheet-support.md [hunk 3]
```diff
@@ -1,24 +1,25 @@
 ---
-title: "Map Style Support | Microsoft Docs"
-ms.custom: ""
-ms.date: "05/26/2020"
-ms.reviewer: ""
-ms.suite: ""
-ms.tgt_pltfrm: ""
-ms.topic: "article"
+title: Map Style Support | Microsoft Docs
+description: Map styling can be applied to a variety of Microsoft map controls, including Bing Maps, Static, Windows, Android, or iOS.
```
Comment (stevemunk) [inline BingMaps/styling/map-style-sheet-support.md:3]:
```suggestion
description: Learn about map styles supported in a variety of Microsoft map controls with a list showing the controls that support it, a link to the Map Style Sheet Editor application and a table showing the minimum version of each control that supports each style version.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Review by stevemunk — APPROVED** [review]

**Pull request merged**